motionplan/metrics.go

```diff
@@ -113,5 +113,19 @@ func L2InputMetric(segment *Segment) float64 {
 	return referenceframe.InputsL2Distance(segment.StartConfiguration, segment.EndConfiguration)
 }
 
+// SquaredNormSegmentMetric is a metric which will return the cartesian distance between the two positions.
+func SquaredNormSegmentMetric(segment *Segment) float64 {
```

Comment:
Did this get moved or did it really not exist before?

Reply:
Did not exist before.

There's a State metric for squared norm to goal, but was not a segment metric for squared norm between two arbitrary things